pkg/shell/topnav.jsx

```diff
@@ -100,6 +102,17 @@ export class TopNav extends React.Component {
         return null;
     }
 
+    handleModeClick = (isSelected, event) => {
+        const theme = event.currentTarget.id;
+        this.setState({ theme: theme });
+        const styleEvent = new CustomEvent("cockpit-style", {
+            detail: {
+                style: theme,
+            }
+        });
+        window.dispatchEvent(styleEvent);
+    }
```

Comment:
These 10 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17914-20221201-104014-14c78ccb-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)